### ISSUE TITLE
Block Outlines become White

### ISSUE THREAD

**Issue by ColinTheShark** [MichaelHillcox/XRay-Mod#159]
**Describe the bug**
Block outlines for XRay'ed blocks become white.

**To Reproduce**
I don't know how to reproduce this reliably. Sometimes (or rather often actually) block outlines just become white. Sometimes dependent on the angle I look, sometimes no matter what.

**Expected behavior**
Block outlines shouldn't appear white, I guess? (Unless set to white, of course)

**Screenshots**
 - [Expected](https://i.imgur.com/M6wvg0o.png)
 - [Actual](https://i.imgur.com/oCujk23.png)
 - [Change in Action](https://files.catbox.moe/va5e15.mp4) (catbox.moe)

**Minecraft Enviorment**
 - Minecraft Version: `1.16.4`
 - XRay Mod Version: `xray-1.16.4-2.6.0`
 - Forge Version: `1.16.4 - 35.0.7`

**Additional context**
[Mods in use](https://i.imgur.com/MKReiZ3.png). Directories at the top are my "backups" of mods from previous MC versions (in case I switch back lol)

**Comment by MichaelHillcox** [comment]
It's likely optifine, try without it

**Comment by ColinTheShark** [comment]
> It's likely optifine, try without it

I've moved Optifine out of my mods folder, booted up Minecraft and tried in a singleplayer world.
Issue persists. All highlighted blocks are white.

**Comment by MichaelHillcox** [comment]
Try without the Map mods, if that still doesn't fix it, try without Neat. If all else fails, i'll look into it

**Comment by ColinTheShark** [comment]
All mods except xray out of the mods folder, Issue is gone. *Well, this doesn't help.*
I've removed one mod after another and restarted MC in between. Only after Baritone was removed, the issue was gone.
I checked with all mods including Optifine, but without Baritone and the issue was gone.
Looks like Baritone's pathfinding/ore finding is making this issue here. I'll see to open an issue there.
